tests/acceptance/features/apiActivities/activitiesByFileId.feature

```diff
@@ -0,0 +1,499 @@
+Feature: check activities
+  As a user
+  I want to check who made which changes to files using file-id
+  So that I can track modifications
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario: check add activity for a file using file-id
+    Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile.txt"
+    And we save it into "FILEID"
+    And user "Alice" has created folder "/New Folder"
+    And user "Alice" has copied a file "/textfile.txt" into "New Folder" inside space "Personal" using file-id path "/dav/spaces/<<FILEID>>"
```

Comment:
this should be changed to be similar with:
https://github.com/owncloud/ocis/blob/7bff0f3f3298780f7f0a86110ef33e4c7676e589/tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature#L15

Reply:
@saw-jan You mean to test for both endpoints?:
```
Examples:
      | dav-path |
      | /remote.php/dav/spaces/<<FILEID>> |
      | /dav/spaces/<<FILEID> |
```